cypress/integration/AdminSideCMSTest.feature

```diff
@@ -0,0 +1,115 @@
+Feature: As an admin, I want to be able to login to CMS and use main menu bar for any configurations
+
+    Background: Login
+        Given user is navigating to "/user/login"
+        When user enters credentials
+        And user clicks "Log in" button
+
+    Scenario: user is able to login to cms
+        When user is logged in the user name is displayed in the toolbar
```

Comment:
Can you change this to also test that it is logged in under expected user?
`When user is logged in and the user name "admin" is displayed in the toolbar`

Reply:
done